backend/Watchdog.Core/Watchdog.Core.API/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -34,7 +33,6 @@ public static void RegisterCustomServices(this IServiceCollection services, ICon
             services.AddTransient<IIssueService, IssueService>();
             services.AddTransient<IUserService, UserService>();
             services.AddTransient<ITileService, TileService>();
-            services.AddTransient<ITeamService, TeamService>();
```

Comment:
Why do you delete TeamService?

Reply:
There were dublicates 
![image](https://user-images.githubusercontent.com/75535425/129485657-f18ca9fd-4b2f-4d53-b6d5-54fc05ad4ad7.png)